recipe/recipe.yaml

```diff
@@ -22,65 +17,77 @@ source:
       url: https://pypi.org/packages/source/p/polars-u64-idx/polars_u64_idx-${{ version }}.tar.gz
       sha256: 3f8397b7fc77581fb39be465fe99ca584e2e8e94b4554ec1b1e5daeddafe4ed0
 
-build:
-  number: 3
-  skip: not (match(python, python_min ~ ".*") and is_abi3 == 'true')
-  python:
-    version_independent: true
+outputs:
+  - package:
+      name: polars
+      version: ${{ version }}
+    build:
+      number: ${{ build }}
+      skip: not (match(python, python_min ~ ".*") and is_abi3 == 'true')
+      python:
+        version_independent: true
+      variant:
+        down_prioritize_variant: ${{ 0 if polars_variant == 'polars-default' else 1}}
```

Comment:
You should be able to make this noarch
Also, I would prefer adding the variant in dome way into the build string

Reply:
> You should be able to make this noarch
Is that just a matter of adding `noarch: python` or does it need other changes?

Reply:
Also added the python pins as suggested by the linter